global.json

```diff
@@ -1,11 +1,6 @@
 {
   "tools": {
-    "dotnet": "8.0.100-rtm.23506.1",
-    "runtimes": {
-      "dotnet": [
-        "$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)"
-      ]
-    }
+    "dotnet": "8.0.100-rtm.23506.1"
```

Comment:
Should this just be 8.0.0 now?

Reply:
No idea. I just left it as it was. Is there some other flow that updates it?